server.py

```diff
@@ -54,75 +57,143 @@ def get_rank(name):
 
 @my_bot.command()
 async def startGen1(*args):
-	channel = my_bot.get_channel("295737335100997632")
+	channel = my_bot.get_channel("<channel-id>")
```

Comment:
will <channel-id> restrict it to a spesific text channel or does this mean it will work in all channels on the discord server where you added the bot?

Reply:
nvm im stupid, ofc you dont want to expose your IDs and Tokens